app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -8,9 +8,9 @@ def alert_edit
       @edit = nil
       @alert = session[:edit][:alert_id] ? MiqAlert.find_by_id(session[:edit][:alert_id]) : MiqAlert.new
       if @alert && @alert.id.blank?
-        add_flash(_("Add of new %{models} was cancelled by the user") % {:models => ui_lookup(:model => "MiqAlert")})
+        add_flash(_("Add of new Alert was cancelled by the user"))
```

Comment:
`:models` key here and below was confusing but what matters is ui_lookup's argument — this was and remains singular "Alert".